### DISCUSSION TITLE
Removing schema and put it on requestBody (openapi)

### DISCUSSION THREAD

**Discussion by xgorn** [fastapi/fastapi#10028] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [ ] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [ ] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
async def validator(api_key: str = Form(...)):
    pass

@bypass.post('/ouo')
async def ouo(request: Request, url: str = Form(...), validate: dict = Depends(validator)) -> JSONResponse:
    pass
```

### Description

Well... while using swagger docs i found out that the `Form(...)` will actually create a schema with some ugly namespace like `Body_etc_post` and i don't really need that, and i'm looking for a way to remove that schema and just put all the form data inside the openapi requestBody without using any refs just like the example below.

```
"requestBody": {
          "required": true,
          "content": {
            "application/x-www-form-urlencoded": {
              "schema": {
                "type": "object",
                "properties": {
                  "api_key": {
                    "type": "string",
                    "example": "api-key"
                  },
                  "url": {
                    "type": "string",
                    "example": "https://ouo.io/4ZuwGMc"
                  }
                }
              }
            }
          }
        },
```

Hopefully anyone can help me with this.

### Operating System

Windows

### FastAPI Version

0.100.1

### Pydantic Version

1.10.4

### Python Version

Python 3.9.10

**Comment by AlexanderPodorov** [comment]
@X-Gorn , see https://github.com/tiangolo/fastapi/discussions/10023. I have similar issue as well. However I'm suggesting to fix that another way (wrap all `Form`, `File`, etc. into a custom class and use this class name), and the suggested workaround won't work for your use case unfortunately.

**Comment by YuriiMotov** [comment, marked as answer]
I think it's possible to adapt the approach from this response to achieve what you need: https://github.com/fastapi/fastapi/discussions/11859#discussioncomment-13198728
